server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -218,15 +221,50 @@ private TrTag renderTIRow(Account ti) {
         .with(renderStatusCell(ti));
   }
 
-  private TrTag renderApplicantRow(Account applicant) {
+  private TrTag renderApplicantRow(
+      Account applicant, TrustedIntermediaryGroup tiGroup, Http.Request request) {
     return tr().withClasses(
             ReferenceClasses.ADMIN_QUESTION_TABLE_ROW,
             Styles.BORDER_B,
             Styles.BORDER_GRAY_300,
             StyleUtils.even(Styles.BG_GRAY_100))
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant));
+        .with(renderActionsCell(applicant))
+        .with(renderDateOfBirthCell(applicant, tiGroup, request));
+  }
+
+  private TdTag renderDateOfBirthCell(
+      Account account, TrustedIntermediaryGroup tiGroup, Http.Request request) {
+
+    Optional<Applicant> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
+    }
+    Optional<LocalDate> current_DOB = newestApplicant.get().getApplicantData().getDateOfBirth();
+    // for cases where DOB is already present, display the Date of Birth
+    // for other cases, show the form to update the DOB.
+    // Note: the update can be done only once.
```

Comment:
question (blocking): why?

Reply:
for a given applicant, the DOB is unique and can be updated when it isn't present. If it were so, searching by DOB will a difficult--> If T1 edits A1's DOB, and T2 edit A1's DOB again, T1 and T2 will have difficulty searching for A1's DOB.. DOB is should be updated once. I can check if Elise on this.

Reply:
Checked with Elise - we want this to be editable always (whether DOB is present or not)